### PR TITLE
Pods 2943

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: app/identifiers/register/trustees/individual/TrusteeNameId.scala [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ */
+
+package identifiers.register.trustees.individual
+
+import identifiers._
+import identifiers.register.trustees.TrusteesId
+import models.person.PersonName
+import play.api.libs.json.JsPath
+
+case class TrusteeNameId(index: Int) extends TypedIdentifier[PersonName] {
```
Comment (amaalali) [inline app/identifiers/register/trustees/individual/TrusteeNameId.scala]: There seems to be a [`TrusteeDetailsId`](https://github.com/hmrc/pensions-scheme-frontend/blob/master/app/identifiers/register/trustees/individual/TrusteeDetailsId.scala) is this the same as that?

Comment (grantrobson) [reply app/identifiers/register/trustees/individual/TrusteeNameId.scala]: One's the old ID, trusteenameid is the new one with only name (not date)

Comment (amaalali) [reply app/identifiers/register/trustees/individual/TrusteeNameId.scala]: Do you mean the inner type of the TypedIdentifier?

Comment (grantrobson) [reply app/identifiers/register/trustees/individual/TrusteeNameId.scala]: No, there are 2 ids, one for the new name page and one for the old one which includes date of birth field as well I believe.

**Pull request merged**